workflow/scripts/postprocess_diffexp.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+
+
+def process_columns(df):
+    """compute confidence interval for every column starting with b_"""
+    matching_columns = [col for col in df.columns if col.startswith(
+        'b_') and not col.endswith('_se')]
+
+    for col in matching_columns:
+        df[f"{col}_lower"] = df[col] - df[f"{col}_se"]
+        df[f"{col}_upper"] = df[col] + df[f"{col}_se"]
+
+    return df, matching_columns
+
+
+def sort_columns(df, matching_columns):
+    b_column_order = [f"{prefix}{suffix}" for prefix in matching_columns for suffix in [
+        '_lower', '', '_upper', '_se']]
+    other_columns = [col for col in df.columns if not col.startswith('b_')]
+    return df[other_columns + b_column_order]
+
+
+def sort_rows(df):
+    """Sort DataFrame by the absolute value of signed_p_value of primary variable in ascending order."""
+    signed_pi_start = 'signed_pi_value_' + \
+        snakemake.params['model']['primary_variable']
```

Comment:
I think an f-string would be more succinct and probably more idiomatic, as well:

```suggestion
    signed_pi_start = f"signed_pi_value_{snakemake.params['model']['primary_variable']}"
```

Reply:
From what I can see, this is not yet addressed? You can actually directly hit the `Commit suggestion` button on the actual suggestion box to immediately submit it as a commit.